impactlab_tools/utils/paralog.py

```diff
@@ -69,8 +69,12 @@ def __init__(self, jobname, jobtitle, logdir, timeout, exclusive_jobnames=None):
 
     def __del__(self):
         """Usually not used; allow std output to go back to its previous stream."""
-        sys.stdout.close()
-        sys.stdout = self.sys_stdout
+        
+        if sys is not None:
```

Comment:
I think we should also check that this is a `DoubleLogger`. Otherwise, someone else may have replaced it and not want it to be overwritten.